internal/security/secretstore/testdata/kong-admin-config.template.yml

```diff
@@ -0,0 +1,84 @@
+_transform: false
```

Comment:
Why do we need another copy for testing? Can't is just use something like ../../../cmd/security-secretstore-setup/res/kong-admin-config.template.yml ?

Reply:
This file has been removed.

Reply:
There is a separate copy in order to test incorrect configurations. I guess I didn't build those tests in - so I'll build those in and make this a little clearer.

Reply:
I was incorrect in my statement above. You are correct that we can utilize the existing template and don't need to copy - there is no instance where we parse the template file. It uses what is there and replaces the necessary text. I've updated the test and removed the duplicate template.